src/adapter.js

```diff
@@ -0,0 +1,24 @@
+function Adapter(code) {
```

Comment:
Does this replace `src/adapters/adapter.js`? I don't see that file being deleted... but they're duplicates.